lib/master.js

```diff
@@ -91,6 +97,11 @@ class Master extends BaseService {
             .then((serviceReturns) => [serviceReturns]);
         }
 
+        // Start Prometheus Metrics Endpoint
+        if (this.config.metrics.enable_prometheus) {
+            this.metricsEndpoint = new MetricsEndpoint(this.config.metrics);
+        }
```

Comment:
There're 2 configurations you can run the service with - `num_workers: 0` will not use cluster mode and the whole thing will run within the master process. I think in that case your code will not work, cause the server will not be setup, cause see line 91. 

If I'm correct, we need to fix that and need to add a test.

Reply:
This is true.  Handled in latest commit.